mock/py/mockbuild/backend.py

```diff
@@ -708,6 +708,7 @@ def get_command(mode):
                 except Error as e:
                     if e.resultcode != 11:
```

Comment:
Please document the fact that we accept exit_status 0 and exit_status 11, and why.